pkgs/applications/misc/dwmblocks-async/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitHub, libX11, patches ? [ ] }:
+
+stdenv.mkDerivation {
+  pname = "dwmblocks-async";
+  version = "unstable-2023-04-05";
+
+  src = fetchFromGitHub {
+    owner = "UtkarshVerma";
+    repo = "dwmblocks-async";
+    rev = "27731295335d31966491f55f54912c3b9dd1a7a7";
+    sha256 = "sha256-9VHalp9NM26yQ3A9NgbPFiukxfaKnv/VcKJpQh1lhu4=";
+  };
+
+  inherit patches;
```

Comment:
I dont think this is needed. If you want to add patches you can use `overrideAttrs`.